app/components/Poll/index.tsx

```diff
@@ -51,197 +51,243 @@ type OptionEntityRatio = OptionEntity & {
-                  >
-                    {option.name}
-                  </Button>
-                </Flex>
-              ))}
-          </Flex>
-        )}
-        <div>
-          <div className={styles.moreOptionsLink}>
-            {truncateOptions &&
-              (!poll?.hasAnswered ||
-                !poll?.resultsHidden ||
-                allowedToViewHiddenResults) && (
-                <Flex alignItems="center" justifyContent="center">
-                  <Icon
-                    onClick={toggleTruncate}
-                    name={expanded ? 'chevron-up' : 'chevron-down'}
-                    size={20}
-                  />
-                </Flex>
-              )}
-          </div>
-          <Flex justifyContent="space-between">
+        </Link>
+      </Flex>
+      <Flex
+        column
+        className={styles.contentWrapper}
+        style={{
+          height: alwaysOpen ? `auto` : expanded ? `${expandedHeight}px` : '0',
+        }}
+      >
+        <div ref={optionRef}>
+          {canAnswer && (
+            <VoteOpen details={details} poll={poll} options={options} />
+          )}
+          {!canAnswer && showResults && (
+            <VoteResults
+              details={details}
+              poll={poll}
+              options={options}
+              resultsHidden={resultsHidden}
+            />
+          )}
+          {!canAnswer && !showResults && (
+            <VoteHidden details={details} poll={poll} />
+          )}
+
+          <Flex
+            alignItems="center"
+            justifyContent="center"
+            className={styles.registrationCount}
+          >
+            <Tooltip content="Avstemningen er anonym.">
+              <Icon
+                name="information-circle-outline"
+                size={17}
+                style={{ cursor: 'pointer', margin: '0 5px' }}
+              />
+            </Tooltip>
             <span>
-              <span className={styles.totalVotes}>{poll?.totalVotes}</span>{' '}
-              {poll?.totalVotes === 1 ? 'stemme' : 'stemmer'}
+              <span className={styles.totalVotes}>{totalVotes}</span>{' '}
+              {totalVotes === 1 ? 'stemme' : 'stemmer'}
             </span>
-            {poll?.hasAnswered && !showResults && (
-              <span className="secondaryFontColor">Resultatet er skjult</span>
-            )}
           </Flex>
         </div>
       </Flex>
-    </Card>
+      <Flex
+        alignItems="center"
+        justifyContent="center"
+        className={styles.bottomBar}
+        style={{ cursor: alwaysOpen ? '' : 'pointer' }}
+        onClick={alwaysOpen ? undefined : toggleExpanded}
+      >
+        {!alwaysOpen && (
+          <Icon
+            className={styles.arrowIcon}
+            size={26}
+            name={expanded ? 'chevron-up' : 'chevron-down'}
+          />
+        )}
+      </Flex>
+    </Flex>
   );
 };
 
 export default Poll;
+
+type VoteOpenProps = {
+  poll: PollEntity;
```

Comment:
```suggestion
  poll: Poll;
```
I suppose we should migrate over to the more correct `Poll` type in `app/store/models/Poll`